src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -151973,27 +151973,18 @@ public override void M1<T>(C<System.Nullable<T?> x)
 class C<T> {}
 ");
             comp.VerifyDiagnostics(
```

Comment:
what's going on here?  What was `C<System.Nullable<T?> x` previously parsed as?  And what is it now parsed as?

Reply:
If not mistaken, this was previously being parsed as having a missing comma before the `x`, with the type argument list closing on the `)` with a missing `>` token and a missing identifier on the parameter. Therefore, it was implying
```csharp
public override void M1<T>(C<System.Nullable<T?>, x> @identifier)
```

We are now parsing this with `x` being the identifier name, since we look ahead, and thus expect the following:
```
public override void M1<T>(C<System.Nullable<T?>> x)
```